full-service/src/json_rpc/json_rpc_request.rs

```diff
@@ -66,11 +66,12 @@ pub enum JsonCommandRequest {
     },
     build_and_submit_transaction {
         account_id: String,
-        addresses_and_values: Option<Vec<(String, String)>>,
+        addresses_and_amounts: Option<Vec<(String, Amount)>>,
```

Comment:
To make this backward compatible, we can leave the old value, and if it is None, use the new value, and if it is populated, use it.

```suggestion
        # Multi-token Support
        addresses_and_amounts: Option<Vec<(String, Amount)>>,
        # MOB-only
        addresses_and_values: Option<Vec<(String, String)>>,
```